pkg/syncer/metadatasyncer.go

```diff
@@ -41,6 +44,32 @@ func NewInformer() *MetadataSyncInformer {
 	return &MetadataSyncInformer{}
 }
 
+// getFullSyncIntervalInMin return the FullSyncInterval
+// If enviroment variable FULL_SYNC_INTERVAL_MINUTES is set and valid,
+// return the interval value read from enviroment variable
+// otherwise, use the default value 30 minutes
+func getFullSyncIntervalInMin() int {
+	fullSyncIntervalInMin := defaultFullSyncIntervalInMin
+	if v := os.Getenv("FULL_SYNC_INTERVAL_MINUTES"); v != "" {
+		if value, err := strconv.Atoi(v); err == nil {
+			if value <= 0 {
```

Comment:
lets combine `if value <= 0 {` and `value > defaultFullSyncIntervalInMin` and log invalid range message.